src/test/java/co/omise/live/LiveChargeRequestTest.java

```diff
@@ -0,0 +1,323 @@
+package co.omise.live;
+
+import co.omise.Client;
+import co.omise.ClientException;
+import co.omise.models.*;
+import co.omise.requests.Request;
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNotNull;
+
+public class LiveChargeRequestTest extends BaseLiveTest {
+    private Client client;
+
+    @Before
+    public void setup() throws ClientException {
+        client = new Client(getPublicKey(), getSecretKey());
```

Comment:
Shall we move `Client` initialize to `BaseLiveTest` to re-use in other test classes? 😁

Reply:
Yeah sure, have you done it anywhere?

Reply:
I have done that in my PR.

Reply:
Okk I'll just copy that

Reply:
Done